lib/tapioca/compilers/sorbet.rb

```diff
@@ -26,7 +26,12 @@ def run(*args)
 
         sig { returns(String) }
         def sorbet_path
-          SORBET.to_s.shellescape
+          custom_sorbet_path = ENV["TPC_SORBET_EXE"]
+          if custom_sorbet_path.nil? || custom_sorbet_path.empty?
+            SORBET.to_s.shellescape
+          else
+            custom_sorbet_path.shellescape
+          end
```

Comment:
```suggestion
          sorbet_path = ENV.fetch("TAPIOCA_SORBET_EXE", SORBET)
          sorbet_path = SORBET if sorbet_path.empty?
          sorbet_path.to_s.shellescape
```
would be much more succinct.